README.adoc

```diff
@@ -83,15 +67,75 @@ include::{common-includes}/gitclone.adoc[]
 === Try what you'll build
 
 The `finish` directory in the root of this guide contains the finished application. Give it a try before you proceed.
+Run the following command to test that Maven Wrapper is installed:
+
+include::{common-includes}/os-tabs.adoc[]
+
+[.tab_content.windows_section]
+--
+[role='command']
+```
+mvnw.cmd -v
+```
+--
+
+[.tab_content.mac_section]
+--
+[role='command']
+```
+./mvnw -v
+```
+--
+
+[.tab_content.linux_section]
+--
+[role='command']
+```
+./mvnw -v
+```
+--
+
+If Maven Wrapper is installed properly, you see information about the Maven installation similar to the following example:
+
+[source, role="no_copy"]
+----
+Apache Maven 3.9.6 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
+Maven home: /Applications/Maven/apache-maven-3.9.6
+Java version: 11.0.12, vendor: International Business Machines Corporation, runtime: /Library/Java/JavaVirtualMachines/ibm-semeru-open-11.jdk/Contents/Home
+Default locale: en_US, platform encoding: UTF-8
+OS name: "mac os x", version: "11.6", arch: "x86_64", family: "mac"
+----
 
 // tag::runCommand[]
 To try out the application, first go to the `finish` directory and run Maven with the `liberty:run` goal to build the application and deploy it to Open Liberty:
+include::{common-includes}/os-tabs.adoc[]
+
+[.tab_content.windows_section]
+--
+[role='command']
+```
+cd finish
```

Comment:
We are already in the `finish` directory from the previous step. I would remove it.